test/dotnet-list-package.Tests/GivenDotnetListPackage.cs

```diff
@@ -349,5 +349,51 @@ public void ItRunsInCurrentDirectoryWithPoundInPath()
                 .Should()
                 .Pass();
         }
+
+        [Fact]
+        public void ItRecognizesRelativePathsForAProject()
+        {
+            var testAssetName = "TestAppSimple";
+            var testAsset = _testAssetsManager
+                .CopyTestAsset(testAssetName)
+                .WithSource();
+
+            var projectDirectory = testAsset.Path;
+
+            new RestoreCommand(testAsset)
+                .Execute()
+                .Should()
+                .Pass();
+
+            new ListPackageCommand(Log)
+                .WithProject("TestAppSimple.csproj")
+                .WithWorkingDirectory(projectDirectory)
```

Comment:
Can we add a test where the relative path is not just a file within the same folder?

Reply:
Good catch. Will add one more test for the scenario.